src/middlewared/middlewared/plugins/smb.py

```diff
@@ -268,6 +286,30 @@ def key_cp(encoding):
             ] + initial
         }
 
+    @private
+    @periodic(interval=86400)
```

Comment:
What is the point of doing this every day?

Reply:
Checking the running configuration in case users edit via CLI. Thought it's not that impactful to have a periodic check.

Reply:
@anodos325 what CLI?

Reply:
Manual edits of smb running configuration through text editor or `net conf` utility.

Reply:
Do we support this? Why?

Reply:
Do we support this? Why?